apps/admin-gui/src/app/users/pages/user-detail-page/user-settings/user-settings-associated-users/user-settings-associated-users.component.ts

```diff
@@ -37,38 +37,34 @@ export class UserSettingsAssociatedUsersComponent implements OnInit {
 
   ngOnInit(): void {
     this.loading = true;
-
-    this.route.parent.params
-      .subscribe(params => {
-        this.userId = params["userId"];
-        this.userManager.getUsersBySpecificUser(this.userId).subscribe(associatedUsers => {
-          this.associatedUsers = associatedUsers;
-          this.setAuth();
-          this.loading = false;
-        });
-      });
+    this.user = this.entityStorageService.getEntity();
+    this.userManager.getUsersBySpecificUser(this.user.id).subscribe(associatedUsers => {
```

Comment:
also on this component in the html on the buttons remove the "loading" condition from the *ngIf